src/libs/isReportMessageAttachment.js

```diff
@@ -0,0 +1,12 @@
+import CONST from '../CONST';
+
+/**
+ * Check whether a report action is Attachment or not.
+ * Ignore messages containing [Attachment] as the main content. Attachments are actions with only text as [Attachment].
+ *
+ * @param {Object} reportActionMessage report action's message as text and html
+ * @returns {Boolean}
+ */
+export default function isReportMessageAttachment({text, html}) {
+    return text === CONST.ATTACHMENT_MESSAGE_TEXT && html !== CONST.ATTACHMENT_MESSAGE_TEXT;
```

Comment:
What does the second half of this do? Is it checking if someone typed the phrase `[Attachment]`? I guess NAB since it was in the old function too, but out of interest...

Reply:
Tbh, I don't really know. Can't think of a case where the `html` would be equal to `[Attachment]`. Could have been someone found this bug randomly by throwing stuff at a wall, reported it, then we fixed it. Or maybe it's important 😅

Reply:
Ha. If it ain't broke...